examples/perfbench/perfbench.cpp

```diff
@@ -37,14 +37,24 @@
 #pragma warning(disable : 4244)
 #pragma warning(disable : 4305)
 #endif
+#ifdef _WIN32
+#define WINDOWS
+#endif
+#ifdef _WIN64
+#define WINDOWS
+#endif
 
 #include "../timing.h"
 #include <algorithm>
 #include <stdio.h>
 
 #include "perfbench_ispc.h"
 
+#ifdef WINDOWS
+typedef void(__vectorcall FuncType)(float *, int, float *, float *);
```

Comment:
I think defining macro ``CALLING_CONV`` (or something like that) and setting it's value to either empty or ``__vectorcall`` is better options from readablility prospective.

Reply:
Done